tests/Resources/Page/OCPDashboard/Monitoring/Grafana.robot

```diff
@@ -0,0 +1,30 @@
+*** Settings ***
+Documentation    Cointans Grafana Releted keyword
+Library        OperatingSystem
+Library        Process
+Library        SeleniumLibrary
+Library        RequestsLibrary
+
+*** Variables ***
+${SLIDE_MENU_DROPDOWN}          //div[@class="sidemenu-item dropdown"]
+${GRAFANA_SEARCH_BUTTON}        //*[text()="Search"]
+
+
+*** Keywords ***
+Go To Grafana Dashbord Search
+    [Documentation]    Check if Jupyterhub SLI option is available
+    Wait Until Element Is Visible     ${SLIDE_MENU_DROPDOWN}
+    Run Keywords
+    ...    Click Element    ${SLIDE_MENU_DROPDOWN}
+    ...    AND
+    ...    Click Element    ${GRAFANA_SEARCH_BUTTON}
+
+Wait Until Grafana Page Is UP
+    [Arguments]  ${TIMEOUT}
+    [Documentation]  Wait until grafana Page is up and running
+    FOR  ${i}   IN RANGE  ${TIMEOUT}
+        Sleep  1s   msg=Waiting until port forwarding is active
```

Comment:
maybe you could try removing the `s` (seconds) from the Sleep, so you could pass whatever time measure you want (seconds, minutes, etc). otherwise it'd better to specify that the variable `TIMEOUT` only takes numeric values which are intended to be "seconds". 

maybe a easier implementation is to use two args `retries` and `interval between retries` like done in `Wait Until RHODS Uninstallation Is Completed`